install/ansible/istio/vars/main.yml

```diff
@@ -3,15 +3,6 @@ github_url: https://api.github.com/repos
 istio_repo: istio/istio
 default_github_api_headers: {}
 minimum_cluster_version: 1.7.0
```

Comment:
istio-demo only runs on 1.9/1.10 because of the sidecar injector.

Reply:
Great, I will update it